### DISCUSSION TITLE
Cant find Imagick for PHP 8.1?

### DISCUSSION THREAD

**Discussion by zipnfc** [ImageMagick/ImageMagick#7154] [General] [answered]
I have seen posts that Imagick is not compatible with PHP 8.1 as yet and not available on Cpanel as an extension.  Unfortunately we have ported all out software over to 8.1 OK and thi sis the final thing to get working.

Can someone confirm if this is the case and if a way of running on 8.1 please let us know. We are running on AlmaLinux 9

Thanks in advance

**Comment by dlemstra** [comment, marked as answer]
This is not the right spot to ask that question you will need to go here: https://github.com/Imagick/imagick

**Reply by zipnfc** [reply]
Thank you

**Comment by fmw42** [comment]
Perhaps at https://www.php.net/downloads

**Reply by zipnfc** [reply]
That is for downloading PHP itself - but thank you

**Reply by fmw42** [reply]
Sorry, see https://pecl.php.net/package/imagick
